libraries/theme/src/main/kotlin/io/element/android/libraries/theme/ElementTheme.kt

```diff
@@ -115,6 +116,7 @@ fun ElementTheme(
     }
     CompositionLocalProvider(
         LocalCompoundColors provides currentCompoundColor,
+        LocalContentColor provides colorScheme.onSurface,
```

Comment:
Should it be `colorScheme.primary` instead?

Reply:
Both semantic tokens point to the same color token, and I think `onSurface` matches the default behavior better, as this `LocalContentColor` is usually provided by a parent `Surface` component, but I can change it if needed.